backends/occa/ceed-occa-restrict.okl

```diff
@@ -1,11 +1,26 @@
+// Copyright (c) 2017, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-734707. All Rights
+// reserved. See files LICENSE and NOTICE for details.
+//
+// This file is part of CEED, a collection of benchmarks, miniapps, software
+// libraries and APIs for efficient high-order finite element and spectral
+// element discretizations for exascale applications. For more information and
+// source code availability see http://github.com/ceed.
+//
+// The CEED research is supported by the Exascale Computing Project 17-SC-20-SC,
+// a collaborative effort of two U.S. Department of Energy organizations (Office
+// of Science and the National Nuclear Security Administration) responsible for
+// the planning and preparation of a capable exascale ecosystem, including
+// software, applications, hardware, advanced system engineering and early
+// testbed platforms, in support of the nation's exascale computing imperative.
+
 // *****************************************************************************
 kernel void kRestrict0(const int *indices,
                        const double* uu,
                        double* vv) {
   for (int i=0; i<esize; i++; tile(TILE_SIZE)){
-    if (i < esize){
-      vv[i] = uu[indices[i]];
-    }
+    if (i >= esize) continue;
```

Comment:
How could this be true when the controlling expression of the for loop says it does not happen? Or is this an occa thing related to tiling/batching?